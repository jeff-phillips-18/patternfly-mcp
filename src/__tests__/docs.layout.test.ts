import * as docsLayout from '../docs.layout';

describe('docsLayout', () => {
  it('should return specific properties', () => {
    expect(docsLayout).toMatchSnapshot();
  });
});

