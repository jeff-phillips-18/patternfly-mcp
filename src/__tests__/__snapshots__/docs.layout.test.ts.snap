// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`docsLayout should return specific properties 1`] = `
{
  "LAYOUT_DOCS": [
    "[@patternfly/Bullseye - Design Guidelines](https://raw.githubusercontent.com/patternfly/patternfly-org/refs/heads/main/packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/bullseye.md)",
    "[@patternfly/Flex - Design Guidelines](https://raw.githubusercontent.com/patternfly/patternfly-org/refs/heads/main/packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/flex.md)",
    "[@patternfly/Gallery - Design Guidelines](https://raw.githubusercontent.com/patternfly/patternfly-org/refs/heads/main/packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/gallery.md)",
    "[@patternfly/Grid - Design Guidelines](https://raw.githubusercontent.com/patternfly/patternfly-org/refs/heads/main/packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/grid.md)",
    "[@patternfly/Level - Design Guidelines](https://raw.githubusercontent.com/patternfly/patternfly-org/refs/heads/main/packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/level.md)",
    "[@patternfly/Split - Design Guidelines](https://raw.githubusercontent.com/patternfly/patternfly-org/refs/heads/main/packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/split.md)",
    "[@patternfly/Stack - Design Guidelines](https://raw.githubusercontent.com/patternfly/patternfly-org/refs/heads/main/packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/stack.md)",
  ],
}
`;
